packages/markitup/src/markitup/converters/_pdf_converter.py

```
@@ -56,20 +55,27 @@ def convert(
             table_strategy='lines',
             extract_words=True,
             embed_images=False,
             page_chunks=True)
-        
+
         final_chunk_list = []
         for idx, pdf_dict in enumerate(pdf_dict_list):
             md_content = pdf_dict['text']
             words_tuple_list = pdf_dict['words']
 
+            # 处理无文本内容的页面：使用整页截图作为替代
```

Ruff (0.14.5)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)